WooCommerce/WooCommerceTests/Mocks/MockFeatureFlagService.swift

```diff
@@ -1,30 +1,31 @@
 @testable import WooCommerce
 import Experiments
 
-struct MockFeatureFlagService: FeatureFlagService {
-    private let isInboxOn: Bool
-    private let isShowInboxCTAEnabled: Bool
-    private let isUpdateOrderOptimisticallyOn: Bool
-    private let shippingLabelsOnboardingM1: Bool
-    private let isDomainSettingsEnabled: Bool
-    private let isSupportRequestEnabled: Bool
-    private let jetpackSetupWithApplicationPassword: Bool
-    private let betterCustomerSelectionInOrder: Bool
-    private let productBundlesInOrderForm: Bool
-    private let isScanToUpdateInventoryEnabled: Bool
-    private let isBackendReceiptsEnabled: Bool
-    private let sideBySideViewForOrderForm: Bool
-    private let isSubscriptionsInOrderCreationCustomersEnabled: Bool
-    private let isPointOfSaleEnabled: Bool
-    private let googleAdsCampaignCreationOnWebView: Bool
-    private let blazeEvergreenCampaigns: Bool
-    private let blazeCampaignObjective: Bool
-    private let revampedShippingLabelCreation: Bool
-    private let viewEditCustomFieldsInProductsAndOrders: Bool
-    private let favoriteProducts: Bool
-    private let paymentsOnboardingInPointOfSale: Bool
-    private let isProductGlobalUniqueIdentifierSupported: Bool
-    private let isSendReceiptAfterPaymentEnabled: Bool
+class MockFeatureFlagService: FeatureFlagService {
+    var isInboxOn: Bool
+    var isShowInboxCTAEnabled: Bool
+    var isUpdateOrderOptimisticallyOn: Bool
+    var shippingLabelsOnboardingM1: Bool
+    var isDomainSettingsEnabled: Bool
+    var isSupportRequestEnabled: Bool
+    var jetpackSetupWithApplicationPassword: Bool
+    var betterCustomerSelectionInOrder: Bool
+    var productBundlesInOrderForm: Bool
+    var isScanToUpdateInventoryEnabled: Bool
+    var isBackendReceiptsEnabled: Bool
+    var sideBySideViewForOrderForm: Bool
+    var isSubscriptionsInOrderCreationCustomersEnabled: Bool
+    var isPointOfSaleEnabled: Bool
+    var googleAdsCampaignCreationOnWebView: Bool
+    var blazeEvergreenCampaigns: Bool
+    var blazeCampaignObjective: Bool
+    var revampedShippingLabelCreation: Bool
+    var viewEditCustomFieldsInProductsAndOrders: Bool
+    var favoriteProducts: Bool
+    var paymentsOnboardingInPointOfSale: Bool
+    var isProductGlobalUniqueIdentifierSupported: Bool
+    var isSendReceiptAfterPaymentEnabled: Bool
+    var tapToPayEducation: Bool = false
```

Comment:
nit: maybe can use the default value from init for consistency?
```suggestion
    var tapToPayEducation: Bool
```